### PR TITLE
Convert WidgetSample html page into package

### PR DESCRIPTION
**Pull request by jasonlemay**
Actions for the ValueListBox dropdown into /#!/widget-sample are not logged ... otherwise everything looks like it's working!

### CODE REVIEW

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/chosensample/ChosensampleView.ui.xml [hunk 1]
```diff
@@ -34,11 +34,6 @@
                     </div>
                 </div>
             </header>
-
```
Comment (olafleur) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/chosensample/ChosensampleView.ui.xml]: Why did you remove this warning?

File: sample/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html>
 <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+    <meta name="viewport" coﬂntent="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```
Comment (olafleur) [inline sample/src/main/webapp/index.html]: ?

**Comment by olafleur** [comment]
LGTM

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java [hunk 3]
```diff
@@ -72,7 +72,7 @@ public MyEventHandlers(String id) {
 
         private void log(String eventName, String additional) {
             $("#log").append(
-                    "<span class=\"log-line\">" + eventName + " fired by <em>" + elementId + "</em> "
+                    "<span class=\"" + res.style().log_line() + "\">" + eventName + " fired by <em>" + elementId + "</em> "
```
Comment (olafleur) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java]: too long line

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java [hunk 4]
```diff
@@ -131,21 +130,38 @@ public void onValueChange(ValueChangeEvent valueChangeEvent) {
     ChosenValueListBox<Choices> chosenValueListBox;
     @UiField(provided = true)
     MultipleChosenValueListBox<Choices> multipleChosenValueListBox;
+    @UiField
+    static AppResources res;
```
Comment (olafleur) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java]: Do you need that to be static? If yes, it should be before the other UiFields.

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/chosensample/ChosensampleView.ui.xml (hunk 1 above)
Comment (jasonlemay) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/chosensample/ChosensampleView.ui.xml]: Because there's already a "you don't have JS" warning into the main page (index.html). I'm pretty sure anyway that this page would be executed with JS and therefore if the user doesnt have JS turned on, it will not be able to see this warning nested into the app, running with JS :)

**Comment by jasonlemay** [comment]
@meriouma what d'you think?

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java [hunk 5]
```diff
@@ -72,8 +72,10 @@ public MyEventHandlers(String id) {
 
         private void log(String eventName, String additional) {
             $("#log").append(
-                    "<span class=\"log-line\">" + eventName + " fired by <em>" + elementId + "</em> "
-                            + additional + "</span>").scrollTop($("#log").get(0).getScrollHeight());
+                    "<span class=\"" + res.style().log_line() + "\">" +
```
Comment (olafleur) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java]: checkstyle : trailing whitespace

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java [hunk 6]
```diff
@@ -119,10 +119,13 @@ public void onValueChange(ValueChangeEvent valueChangeEvent) {
...
 
-    @UiTemplate("View.ui.xml")
-    interface MyUiBinder extends UiBinder<Widget, WidgetSample> {
+    interface Binder extends UiBinder<Widget, ViewView> {
     }
 
+    private static Binder binder = GWT.create(Binder.class);
+
+    @UiField
+    static AppResources res;
```
Comment (olafleur) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/widgetsample/ViewView.java]: checkstyle : Should be before the GWT.create(...) (package-private before private)

File: sample/src/main/java/com/arcbees/chosen/sample/client/application/ApplicationModule.java [hunk 7]
```diff
@@ -17,12 +17,14 @@
...
 
 import com.arcbees.chosen.sample.client.application.chosensample.ChosensampleModule;
+import com.arcbees.chosen.sample.client.application.widgetsample.WidgetsampleModule;
 import com.gwtplatform.mvp.client.gin.AbstractPresenterModule;
 
 public class ApplicationModule extends AbstractPresenterModule {
     @Override
     protected void configure() {
         install(new ChosensampleModule());
+        install(new WidgetsampleModule());
```
Comment (olafleur) [inline sample/src/main/java/com/arcbees/chosen/sample/client/application/ApplicationModule.java]: I think you can remove that line also.
